autolens/plot/plotters/fit_interferometer_plotters.py

```diff
@@ -0,0 +1,169 @@
+from autoarray.plot.plotters import fit_interferometer_plotters
+from autoarray.plot.plotters import inversion_plotters
+from autogalaxy.plot.mat_wrap import lensing_mat_plot, lensing_include, lensing_visuals
+from autogalaxy.plot.plotters import plane_plotters
+from autolens.plot.plotters import ray_tracing_plotters
+from autolens.fit import fit as f
+
+
+class FitInterferometerPlotter(fit_interferometer_plotters.FitInterferometerPlotter):
+    def __init__(
+        self,
+        fit: f.FitInterferometer,
+        mat_plot_1d: lensing_mat_plot.MatPlot1D = lensing_mat_plot.MatPlot1D(),
+        visuals_1d: lensing_visuals.Visuals1D = lensing_visuals.Visuals1D(),
+        include_1d: lensing_include.Include1D = lensing_include.Include1D(),
+        mat_plot_2d: lensing_mat_plot.MatPlot2D = lensing_mat_plot.MatPlot2D(),
+        visuals_2d: lensing_visuals.Visuals2D = lensing_visuals.Visuals2D(),
+        include_2d: lensing_include.Include2D = lensing_include.Include2D(),
+    ):
+
+        super().__init__(
+            fit=fit,
+            mat_plot_1d=mat_plot_1d,
+            include_1d=include_1d,
+            visuals_1d=visuals_1d,
+            mat_plot_2d=mat_plot_2d,
+            include_2d=include_2d,
+            visuals_2d=visuals_2d,
+        )
+
+    @property
+    def visuals_with_include_2d(self) -> "vis.Visuals2D":
+        """
+        Extracts from a `Structure` attributes that can be plotted and return them in a `Visuals` object.
+
+        Only attributes with `True` entries in the `Include` object are extracted for plotting.
+
+        From an `AbstractStructure` the following attributes can be extracted for plotting:
+
+        - origin: the (y,x) origin of the structure's coordinate system.
+        - mask: the mask of the structure.
+        - border: the border of the structure's mask.
+
+        Parameters
+        ----------
+        structure : abstract_structure.AbstractStructure
+            The structure whose attributes are extracted for plotting.
+
+        Returns
+        -------
+        vis.Visuals2D
+            The collection of attributes that can be plotted by a `Plotter2D` object.
+        """
+
+        visuals_2d = super(FitInterferometerPlotter, self).visuals_with_include_2d
+
+        visuals_2d.mask = None
+
+        return visuals_2d + lensing_visuals.Visuals2D(
+            light_profile_centres=self.extract_2d(
+                "light_profile_centres", self.tracer.planes[0].light_profile_centres
+            ),
+            mass_profile_centres=self.extract_2d(
+                "mass_profile_centres", self.tracer.planes[0].mass_profile_centres
+            ),
+            critical_curves=self.extract_2d(
+                "critical_curves", self.tracer.critical_curves
+            ),
+        )
+
+    @property
+    def tracer(self):
+        return self.fit.tracer
+
+    @property
+    def tracer_plotter(self):
+        return ray_tracing_plotters.TracerPlotter(

```

Comment:
These properties look the same as those implemented in another class - could be moved to a common parent class?

DRY